backend/users/views/user.py

```diff
@@ -47,35 +47,77 @@ def login(self, request, *args, **kwargs):
         except TokenError as e:
             raise InvalidToken(e.args[0])
 
-        token = serializer.validated_data
+        data = serializer.validated_data
 
-        expires = token.current_time + token.lifetime
+        refresh_token = data
+        access_token = data.access_token
```

Comment:
Looking at the `CustomTokenObtainPairSerializer` here, it seems that we could use drf_simplejwt's TokenObtainPairSerializer directly and retrieve the refresh and access token through `data['refresh']` and `data['access']`

Reply:
Here is the code of `TokenObtainPairSerializer`:
```python
class TokenObtainPairSerializer(TokenObtainSerializer):
    @classmethod
    def get_token(cls, user):
        return RefreshToken.for_user(user)

    def validate(self, attrs):
        data = super().validate(attrs)

        refresh = self.get_token(self.user)

        data['refresh'] = str(refresh)
        data['access'] = str(refresh.access_token)

        return data
```
Its validate method returns `str` representation of the tokens.
This is not what we want. We want the direct object.